shared/src/types/ibc/data.rs

```diff
@@ -457,3 +389,76 @@ impl IbcMessage {
         })
     }
 }
+
+/// Receipt for a packet
+#[derive(Clone, Debug)]
+pub struct PacketReceipt(pub Receipt);
+
+impl PacketReceipt {
+    /// Return bytes
+    pub fn as_bytes(&self) -> &[u8] {
+        // same as ibc-go
+        &[1_u8]
+    }
+}
+
+impl Default for PacketReceipt {
+    fn default() -> Self {
+        Self(Receipt::Ok)
+    }
+}
+
+/// Acknowledgement for a packet
+#[derive(Clone, Debug)]
+pub struct PacketAck(pub Acknowledgement);
+
+// TODO temporary type. add a new type for ack to ibc-rs
```

Comment:
should we open issue for these and below too?

Reply:
Thank you for your suggestion!
I'll recheck the implementation of ibc-go and open an issue.